src/Fixer/Alias/NoMixedEchoPrintFixer.php

```diff
@@ -80,12 +80,15 @@ public function isCandidate(Tokens $tokens): bool
         return $tokens->isTokenKindFound($this->candidateTokenType);
     }
 
+    /**
+     * @uses fixPrintToEcho()
+     * @uses fixEchoToPrint()
+     */
     protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
     {
-        $callBack = $this->callBack;
         foreach ($tokens as $index => $token) {
             if ($token->isGivenKind($this->candidateTokenType)) {
-                $this->{$callBack}($tokens, $index);
+                \call_user_func_array($this->callBack, [$tokens, $index]);
```

Comment:
if ($this->candidateTokenType === T_PRINT) {
    $this->fixPrintToEcho($tokens, $index);
} else {
    $this->fixEchoToPrint($tokens, $index);
}

no need to have generic mechanism for any callback if we have only 2 callbacks.
and with this, no need to call method by it's string name